### PR TITLE
Change base definition of AU Core Diagnostic Result Observation to AU Base Diagnostic Result Observation (FHIR-47116)

### PR DESCRIPTION
**Pull request by JayMurdoch**
This PR fixes [FHIR-47116](https://jira.hl7.org/browse/FHIR-47116).

- Changed base definition from Observation to AU Base Diagnostic Result Observation
- Amended General Requirements page - removed timing and instant from Diagnostic Result Observation Observation.effective[x] - [Must Support - Choice of Data Types](https://build.fhir.org/ig/hl7au/au-fhir-core/general-requirements.html#must-support---choice-of-data-types) table as a result of change to Must Support data types from change to base definition  
- change log entry (release 1.0.0 title added, separate PR coming to move necessary entries)

Additional QA issue noted and fixed:
-  from Pathology Result Observation Observation.effective[x] General Requirements page - [Must Support - Choice of Data Types](https://build.fhir.org/ig/hl7au/au-fhir-core/general-requirements.html#must-support---choice-of-data-types). Removed timing and instant from table to reflect MustSupport data types.

### CODE REVIEW

File: input/pagecontent/changes.md [hunk 1]
```diff
@@ -1,6 +1,17 @@
+###  Release 1.0.0
+- Expected Publication date: 2025-01
+- Expected Publication status: Normative
```
Comment (dt-r) [inline input/pagecontent/changes.md:3]: This fix doesn't need a change log entry. Remove changes to changes.md from this PR.

File: input/pagecontent/changes.md [hunk 2]
```diff
@@ -1,6 +1,17 @@
...
+- Based on FHIR version: 4.0.1
+
+This change log documents the significant updates and resolutions implemented from version 1.0.0-ballot to 1.0.0.
+
+#### Changes in this version
+- Made the following changes to AU Core Diagnostic Result Observation:
+  - corrected base profile definition from Observation to AU Base Diagnostic Observation [FHIR-47116](https://jira.hl7.org/browse/FHIR-47116)
```
Comment (dt-r) [inline input/pagecontent/changes.md:10]: make shorter. "- corrected base profile to AU Base Diagnostic Observation [FHIR-47116](https://jira.hl7.org/browse/FHIR-47116)"

File: input/pagecontent/changes.md (hunk 1 above)
Comment (dt-r) [inline input/pagecontent/changes.md:3]: Incorrect. Publication status matches the status of the spec. AU Core R1 will not be normative.

**Review by dt-r — CHANGES_REQUESTED** [review]
1. See inline comments.
2. Missing changed to Must Support documentation for effective[x]

File: input/pagecontent/changes.md (hunk 2 above)
Comment (JayMurdoch) [reply input/pagecontent/changes.md:10]: Amended.

File: input/pagecontent/changes.md (hunk 1 above)
Comment (JayMurdoch) [reply input/pagecontent/changes.md:3]: Publication status changed to "TBC".

Comment (JayMurdoch) [reply input/pagecontent/changes.md]: Confirmed with @brettesler-ext and @dbojicic 'Working Standard'.

**Review by dbojicic — APPROVED** [review]

**Review by dbojicic — APPROVED** [review]

**Pull request merged**